spec/models/user_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject do
+    User.new(name: 'Sam', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Student', post_counter: 0)
+  end
+
+  befor { subject.save }
```

Comment:
Good job setting up the test foundations for this app, however, this method is incorrectly written. It should be `before` not `befor`, this typo is not allowing them to run smoothly. Consider resolving it so the tests can run and after doing that, ensure to run the test with `rspec ./spec/models` for the whole files to run and you'd notice more blockers to the tests. Ensure to fix them before requesting another review.

The Error

<img width="675" alt="tests" src="https://github.com/Progress2002/Blog-App/assets/84930355/bcab2117-68c6-4d46-aea9-c9b69bcad016">